tests/e2e/wdio.conf.js

```diff
@@ -83,10 +84,13 @@ const baseConfig = {
     maxInstances: 1,
     //
     browserName: 'chrome',
+    browserVersion: CHROME_VERSION,
     acceptInsecureCerts: true,
     'goog:chromeOptions': {
-      args: DEBUG ? ['disable-gpu', 'deny-permission-prompts', 'ignore-certificate-errors']: 
-        ['headless', 'disable-gpu', 'deny-permission-prompts', 'ignore-certificate-errors']
+      args: DEBUG ? ['disable-gpu', 'deny-permission-prompts', 'ignore-certificate-errors'] :
+        ['headless', 'disable-gpu', 'deny-permission-prompts', 'ignore-certificate-errors', 'no-sandbox',
+          'window-size=1200,900'],
```

Comment:
This is a bit complicated now with the additional options, particularly because most of the options are the same. Consider pulling out a function or constants to hold these, eg:

```js
const CHROME_OPTIONS = [ ... ];
const CHROME_OPTIONS_DEBUG = CHROME_OPTIONS.concat([ ... ]);
```